### PR TITLE
Wealtho

### PR DESCRIPTION
**Pull request by Thinus01**
In this PR I:

- Created my `WealthO` webpage
- Uses devise for user validation and sign up/log in
- It consists primarly out of 2 pages, category page and transaction page. Both branching out to two more pages
- Category page has a `new category` button that redirects the user to a new page where it allows them to create a new category, that gets saved in the database
- Each category can be interacted with to allow the user to add transactions to it
- Each transaction contributes to the total amount spent in that category (sum amount gets displayed at category) 
- Each user can only see the content that they added, they can not see any categories or transactions made by other users

For a more, in dept show-case, follow this link where I show you how it looks, as well as how it works
https://www.loom.com/share/97cc6eaab9c04a3f90cbbda3b5b1b83d?sid=cc3923b9-a361-456f-9c5c-8f09099c561f
Wealtho

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,120 @@
...
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please consider giving it a star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to thank all code reviewers for making this project better.
```
Comment (DeliceLydia) [inline README.md]: - [ ] Please give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).

File: Gemfile [hunk 2]
```diff
@@ -37,6 +37,8 @@ gem 'jquery-rails'
 
 gem 'sqlite3'
```
Comment (DeliceLydia) [inline Gemfile]: - [ ] Please make sure to use Postgres as the database for the application

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Thinus01, 

Good job so far! There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights
- Used device for authentication
- App has unit tests
- Created video demo


## Suggested changes
_Check the comments under the review._



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @Thinus01, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**